### PR TITLE
Support .NET Core RTM

### PR DESCRIPTION
**Pull request by lvermeulen**
Target frameworks are .net45 and .netstandard 1.1 (which is .net45 and up).

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
-    provider: NuGet
-    on:
-        branch: master
-    server:
-    api_key:
-        secure: RPsb/q5FQhj8t2SkMH4M0RJ0TZSTI7HKJLnDZmvrOYTsiWE8mygz9F9OPThhVlOf
-    skip_symbols: false
+- provider: NuGet
+  api_key:
+    secure: Uft/AgWL0ObDUb6hWLhsftRR1sNhLa5vONUcVa/2KjVAYZApxZD6ckJ+ABFQs3bB
```
Comment (adamreeve) [inline appveyor.yml]: This API key shouldn't be changed

**Comment by adamreeve** [comment]
Thanks, it would be great to support .Net core. I'm on holiday at the moment so will review this properly later.

File: appveyor.yml (hunk 1 above)
Comment (lvermeulen) [inline appveyor.yml]: Correct, I had placed your key in a comment and forgot to replace it. Fixed.

**Comment by adamreeve** [comment]
Sorry for the delay on reviewing this. I still don't have access to a proper internet connection or machine with Visual Studio, so I can't test this out properly myself, but this looks OK to merge. From what I understand the NuGet packages built will continue work fine with .NET 4.5 projects, but no longer support .NET < 4.5, and developing SemVer.NET will now require VisualStudio 2015? And it looks like you've modified the appveyor config so it now only uploads NuGet packages if a new tag is pushed, is that correct? So I'll have to tag a commit with a version number to publish a NuGet release.

**Comment by lvermeulen** [comment]
@adamreeve Yes that's all correct.

**Comment by adamreeve** [comment]
Cool, sounds good :+1:

**Pull request merged**